client/me/security/main.jsx

```diff
@@ -1,43 +1,53 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:me:security:password' );
+import debugFactory from 'debug';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+const debug = debugFactory( 'calypso:me:security:password' );
 
 /**
  * Internal dependencies
  */
-var MeSidebarNavigation = require( 'me/sidebar-navigation' ),
-	Card = require( 'components/card' ),
-	AccountPassword = require( 'me/account-password' ),
-	ReauthRequired = require( 'me/reauth-required' ),
-	twoStepAuthorization = require( 'lib/two-step-authorization' ),
-	SecuritySectionNav = require( 'me/security-section-nav' ),
-	Main = require( 'components/main' );
+import AccountPassword from 'me/account-password';
+import Card from 'components/card';
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import MeSidebarNavigation from 'me/sidebar-navigation';
+import ReauthRequired from 'me/reauth-required';
+import SecuritySectionNav from 'me/security-section-nav';
+import twoStepAuthorization from 'lib/two-step-authorization';
 
-module.exports = React.createClass( {
+class Security extends React.Component {
+	static displayName = 'Security';
```

Comment:
Can we remove `displayName` now?

Reply:
From what I can tell, we mangle variable names but not function names. We should be fine removing `displayName` here and still have the component appear correctly in React devtools, but we use the `displayName` value elsewhere as you've noted in the next comment. I'd opt for the safe route and keep the `displayName` value.

Relevant ESLint rule: [here](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md)